src/main/resources/application.conf

```diff
@@ -0,0 +1,21 @@
+s3 {
+  cleanBucket = "tdr-upload-files-intg"
+  cleanBucket = ${?CLEAN_BUCKET}
+  outputBucket = "tdr-consignment-export-intg"
+  outputBucket = ${?OUTPUT_BUCKET}
+  endpoint = "https://s3.eu-west-2.amazonaws.com/"
+}
+api {
+  url = "https://api.tdr-integration.nationalarchives.gov.uk/graphql"
+  url = ${?API_URL}
+}
+auth {
+  url = "https://auth.tdr-integration.nationalarchives.gov.uk/auth"
+  url = ${?AUTH_URL}
+  clientId = "tdr-backend-checks"
+  clientSecret = ${CLIENT_SECRET}
+}
+efs {
+  rootLocation = "/home/consignment-export/export"
+}
+includeHiddenFiles = false
```

Comment:
Is this setting obsolete now?

Reply:
Yep, good spot. I've removed it and the one from the test application.conf as well.